src/rt/dmain2.d

```diff
@@ -191,6 +274,12 @@ extern (C) int rt_init()
     version (CRuntime_Microsoft)
         init_msvc();
 
+    version(Win64)
+    {{
```

Comment:
why the double braces?

Reply:
introduce a new scope for the local import.